#goal definition
byte LOOKAT = 0
byte PAN_TILT = 1

byte goal_type   # either LOOKAT or PAN_TILT

uint8 priority 	 # [1-255] (action client calls with the same priority cancel each other)

float64 pan_vel      # pan_vel
float64 tilt_vel     # tilt_vel 

# in case of LOOKAT:
geometry_msgs/PointStamped target_point  # use in case of LOOKAT

# in case of PAN_TILT
float64 pan      # use in case of PAN_TILT
float64 tilt     # use in case of PAN_TILT

---
#result
string error
---
#feedback
bool at_setpoint
